src/main/webapp/app/overview/tab-bar/index.ts

```diff
@@ -0,0 +1 @@
+export * from './tab-bar';
```

Comment:
we do not use index.ts files because they can lead to issues. please delete this file